Sources/Common/DIGraph.swift

```diff
@@ -3,11 +3,13 @@ import Foundation
 public class DIGraph {
     public let siteId: SiteId
     public let apiKey: String
+    public let region: Region
     public let sdkConfig: SdkConfig
 
-    public init(siteId: SiteId, apiKey: String, sdkConfig: SdkConfig) {
+    public init(siteId: SiteId, apiKey: String, region: Region, sdkConfig: SdkConfig) {
         self.siteId = siteId
```

Comment:
I like how we have `siteId`, `apiKey`, and `region` as part of `SdkConfig` in Android. Even in iOS, we can't generate a `SdkConfig` object with region, so how about we make them a required member of this class?

Reply:
I agree. Those fields make sense to be in the `SdkConfig`. 

Moving `siteId` and `apiKey` would make this PR larger. I will move `region` to `SdkConfig` and then we can make a new